shared/fs/common/hooks.tsx

```diff
@@ -152,7 +152,7 @@ export const useFsFileContext = (path: Types.Path) => {
   const pathItem = Container.useSelector(state => Constants.getPathItem(state.fs.pathItems, path))
   const [urlError, setUrlError] = React.useState<string>('')
   React.useEffect(() => {
-    urlError && logger.error(`urlError: ${urlError}`)
+    urlError && logger.info(`urlError: ${urlError}`)
```

Comment:
I put in this change after your approval; it's a trivial change but let me know if you don't like it.

Reply:
@mmaxim